File: submissions/sm_011_divyansh/week_03/day_4/session_1/mirrorImage.js
function mirrorImg(input) {
    var letters = 'abcdefghijklmnopqrstuvwxyz';
    var mirror = ''

    for (var i = 0; i < input.length; i++) {
        if (input[i] == input[i].toUpperCase()) {
            lower = input[i].toLowerCase();
            index = letters.indexOf(lower);
            mirror+= letters[25 - index].toUpperCase()
        }
        else {
            index = letters.indexOf(input[i])

            mirror+= letters[25 - index]
        }
    }
    console.log(mirror)
}
mirrorImg('AbCd')   

File: submissions/sm_001_aalind/week_13/day_4/session_1/mirror_image.py
def indexOf(char, string):
    for i in range(len(string)):
        if string[i] == char:
            return i

    return -1

string = input('Enter a word: ')

alphabets_lower = "abcdefghijklmnopqrstuvwxyz"
final_string = ""

for char in string:
    index = indexOf(char.lower(), alphabets_lower)
    mirror_img = alphabets_lower[25-index]

    if char.isupper():
        final_string += mirror_img.upper()
    else:
        final_string += mirror_img.lower()

print(final_string)